NDTensors/src/linearalgebra/linearalgebra.jl

```diff
@@ -489,6 +489,9 @@ end
 # about unpacking Q and L from the A matrix.
 #
 function ql!(A::StridedMatrix{<:LAPACK.BlasFloat})
+  if iscu(A)
```

Comment:
Should we also have `ql` and `ql_positive` move back and forth to CPU to make them work on GPU? Seems like we may as well to make them consistent with `qr_positive`.

We can keep this error message for `ql!` since that strategy doesn't work for in-place operations.